payment.go

```diff
@@ -1,6 +1,89 @@
 package ghipu
 
-import "time"
+import (
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"time"
+)
+
+type PaymentService struct {
+	client *httpClient
+}
+
+func NewPaymentService(secret, receiverID string) *PaymentService {
+	client := httpClient{
+		client: &http.Client{},
```

Comment:
same here, why not use a simple `http.DefaultClient`instead a default construction of a client?